packages/ui/src/components/editor/panels/Files/icon/index.tsx

```diff
@@ -75,7 +78,7 @@ export const FileIcon = ({
   color?: string
 }) => {
   const FallbackIcon = FileIconType[type ?? '']
-
+  thumbnailURL = getState(EngineState).isEditing ? bustURLCache(thumbnailURL) : thumbnailURL
```

Comment:
everywhere this is done in the body of a react component, it will bust the cache every second, meaning that any time it re-renders it will re-request the same file very often

Reply:
What do you think would be a better option here? I'd originally considered making bustURLCache use every 10 seconds as the default hash, but thought that might still result in the old-version cache getting loaded in some cases. I suppose with File Icons loading an old version might be not particularly problematic, should we just remove the cache busting here?

Reply:
i think the ideal solution is to use our CMS generated hashes everywhere, following the cms branch. pending that, i don't see there being a realistic way of scaling this solution well

Reply:
I agree, which is why bustURLCache can also take in a hash instead of using the timestamp default hash, which I assumed would be the CMS-generated hashes once that's in. I wasn't sure which PR would get merged first - if this gets in before the CMS branch, then once that's in someone would just need to pass the CMS hash along to bustURLCache in a few places. If CMS gets in first I can modify this PR to do that CMS hash passing.

The default timestamp hash was to have *something* as a default, and the timestamp seemed unique enough.